src/data/list/basic.lean

```diff
@@ -2694,8 +2694,11 @@ theorem pmap_eq_map_attach {p : α → Prop} (f : Π a, p a → β)
   (l H) : pmap f l H = l.attach.map (λ x, f x.1 (H _ x.2)) :=
 by rw [attach, map_pmap]; exact pmap_congr l (λ _ _ _ _, rfl)
 
+theorem attach_map_coe (l : list α) (f : α → β) : l.attach.map (λ i, f i) = l.map f :=
```

Comment:
```suggestion
@[simp] theorem attach_map_coe (l : list α) (f : α → β) : l.attach.map f = l.map f :=
```

Reply:
There is a coercion hiding in the lambda that makes this spelling invalid.